Sources/SnowManLogic/GameLogic/guessManager.swift

```diff
@@ -0,0 +1,64 @@
+public class GuessManager {
+	public var totalIncorrectGuessesAllowed: Int
+	public var correctGuesses: [String]
+	public var incorrectGuesses: [String]
+	public var guess: String
+
+	public init (totalIncorrectGuessesAllowed: Int) {
+		self.totalIncorrectGuessesAllowed = totalIncorrectGuessesAllowed
+		correctGuesses = []
+		incorrectGuesses = []
+		guess = ""
```

Comment:
This still shouldn't be happening like this. There should be no maintained state of guess, it should just be evaluated when needed.